tests/console/check_migration_features.pm

```diff
@@ -40,20 +40,25 @@ sub check_python2_dropped {
 # SLE-23610: Python3 module
 # test steps:
 # 1) activate the python3 module
-# 2) install the python310 package
-# 3) check python3.10's version which should be 3.10.X
+# 2) install the python31X package
+# 3) check python3.1X's version which should be 3.1X.X
 # 4) check python3's version
-# 5) check python310's lifecycle
+# 5) check python31X's lifecycle
 sub check_python3_module {
     record_info('SLE-23610', 'Check Python3 Module');
     my $OS_VERSION = script_output("grep VERSION_ID /etc/os-release | cut -c13- | head -c -2");
     my $ARCH = get_required_var('ARCH');
     assert_script_run("SUSEConnect -p sle-module-python3/$OS_VERSION/$ARCH", timeout => 180);
-    zypper_call("se python310");
-    zypper_call("in python310");
-    assert_script_run("python3.10 --version | grep Python | grep 3.10.");
+    my $python_version = get_var('PYTHON_VERSIONS', '3.11');
+    my $sub_version = substr($python_version, 2);
```

Comment:
Not a big thing. But why not set `PYTHON_VERSIONS` directly to `311` for simplification?

Reply:
Just follow the python version format to set this.

Reply:
I agree with Yiannis, set PYTHON_VERSIONS to 311 will make the code more simple, then we do not need so complcated code in line 54 & 55 to judge the python 311 package exist or not. WDYT?

Reply:
I prefer to keep use my method. It will be more readable.
For python version, we called it 2.6 or 3.10 but not 311. So I think my method is 'user friendly'.

Reply:
for me it is different three hundred eleven than three point eleven, it is never a good idea to set values artificially for the comfort of the developer, always better to be meaningful from outside.
But I agree this code could be better :), but it would need then a setting `PYTHON3_SUBVERSION` I guess. I will remember for future refactor that something smells here, thanks for the input.

Reply:
I agree that this code is overly complicated. the only difference is '10' or '11'. You most likely just need a condition to set the value of a variable `my $python3_subversion`.

Reply:
Updated description.

Reply:
the description was a minor issue, the code is too complicated and at least 3 contributors thinks the same.
could you please address it?  Basically the suggestion is that there is not need for more code, and just pass '10' or '11'.

Reply:
If only change 10 to 11, but python version changed to 3.22 or 3.12 later, we need change code again.
If use my method,  we can only change setting to update python version later.

Reply:
I don't think the code too complicated. Before we only set 'zypper_call("se python310")'. Now I handled this error message and die the test module if the expected python version cannot get.

Reply:
Following the conclusion of review meeting, use hardcode of python version to avoid complicated code.